src/app/leverage/provider.tsx

```diff
@@ -56,7 +56,7 @@ export interface TokenContext {
   onSelectLeverageType: (type: LeverageType) => void
   onSelectOutputToken: (tokenSymbol: string) => void
   reset: () => void
-  toggleIsMinting: () => void
+  toggleIsMinting: (force?: boolean) => void
```

Comment:
Is this `force` used anywhere?

Reply:
I'm not sure atm, but I wanted to have an option at some point to explicitly set this, because in certain cases toggle will not work